### PR TITLE
Adding favorite colors for users

### PR DESCRIPTION
**Pull request by cmvandrevala**

### CODE REVIEW

**Comment by jjaison26** [comment]
Good idea!

**Comment by znesvacil** [comment]
Wow! Github is awesome

**Comment by Narwaffle22** [comment]
how very interesting

File: my_favorite_color.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Red
```
Comment (xsabreusx) [inline my_favorite_color.txt:1]: Incredible your favorite color is red!

Comment (VenKrish3) [inline my_favorite_color.txt:1]: Red is a good color to like 👍, I like red too.

**Review by xsabreusx — CHANGES_REQUESTED** [review]
Please leave your real favorite color!

**Comment by Ali123-mir** [comment]
Look's good..

**Review by VenKrish3 — APPROVED** [review]
go ahead

File: my_favorite_color.txt
Comment (Narwaffle22) [inline my_favorite_color.txt:1]: woah

**Review by Narwaffle22 — COMMENTED** [review]
is good

**Pull request merged**